samples/msal-browser-samples/VanillaJSTestApp2.0/app/multi-tenant/authConfig.js

```diff
@@ -0,0 +1,67 @@
+const homeTenant = "72f988bf-86f1-41af-91ab-2d7cd011db47";
```

Comment:
Do we want the hardcoded tenants and clientIds in here?

Reply:
I'm actually removing this sample, after talking to @tnorling it's better to add this scenario to the customizable-e2e-sample and add some tests while I'm at it.